app/services/preserved_object_handler_results.rb

```diff
@@ -88,12 +98,32 @@ def result_hash(code, msg_args=nil)
   # result_array = [result1, result2]
   # result1 = {response_code => msg}
   # result2 = {response_code => msg}
-  def log_results
+  def report_results
```

Comment:
this change should have some tests, but since I already wrote some of them, you can leave that to me.